### ISSUE TITLE
Custom Log Collection, Reports and Signatures

### ISSUE THREAD

**Issue by neezy** [cuckoosandbox/cuckoo#1903]
First of all, thank you for creating such a beautiful peace of automation for malware analysis.  I have set this up and it works great!.

Now, I am trying to understand how I can customize it to add additional content to the reports.  What I would like to do is be able to collect a particular Symantec SEP log and show that data under signatures if there is anything in the log.  The reason for doing this is to show that we have custom behavioral signatures that were detected from the sample running so we don't have to create new ones, otherwise if nothing shows up in the logs, we then know we do not have detection and need to create one.  

I basically want to make it into not only a malicious detonator to get IOCs, but also a testing platform.  In order to do that, I need to be pointed in the right direction on what changes in the code I would need to make.

In summary, I need to:

1. Collect/Parse/Analyze logs that Cuckoo does not do natively 
2. Writing a custom signature to pull that data into a report and maybe make the log downloadable as well. 

Any help on this matter is most appreciated!

**Comment by doomedraven** [comment]
you have answer to all that in documentation ;)

**Comment by jbremer** [comment]
All of that is possible, but requires a number of steps. One should be able to implement this based on looking at our code base & documentation and existing processing modules / plugins / Signatures / etc.
If this is something that you'd like to use in a business setting and you don't have the time to implement it yourself, please feel free to take a look at our enterprise options.

**Comment by neezy** [comment]
Thanks,  I will look and check out the code.  I was hoping to get an example since I don't have a whole lot of time to dedicate to modifying it, but I understand from a business standpoint.  I'll try to find some time to figure it out.
